docs/infrastructure/native-image.md

```diff
@@ -224,6 +224,9 @@ following:
     image debugger_
   - using `native,-ls` disables support for _language server_ in the generated
     binary
+  - using `native,-ms` **opts-out** for `Standard.Microsoft` library being
```

Comment:
8af9c47f2f4d6eade5602b47a672c60c98fe4e7f provides a way to **opt-out** of `Standard.Microsoft` being compiled into _native image_. Use:
```
enso$ ENSO_LAUNCHER=native,-ms sbt
enso$ sbt buildEngineDistribution
enso$ sbt runEngineDistribution --env ENSO_SQLSERVER_DATABASE=tempdb --run test/Microsoft_Tests
```
to check how the **opt-out** would behave.